transport/npipe.go

```diff
@@ -0,0 +1,43 @@
+package transport
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"net/url"
+)
+
+func DefaultWindowsTransport() *Transport {
+	t, _ := NpipeTransport("//./pipe/docker_engine")
```

Comment:
I'm not sure why I ignored errors on the unix socket version of this.
Seems like a mistake.

Can we propagate the error? Bonus points if you want to fix the unix one too 👼